Other/EternalDragonDrops.cs

```diff
@@ -0,0 +1,71 @@
+//cs_include Scripts/CoreBots.cs
+
+using Skua.Core.Interfaces;
+using Skua.Core.Options;
+
+public class EternalDragonDrops
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+
+    public List<IOption> Options = new List<IOption>()
+    {
+        CoreBots.Instance.SkipOptions,
+        new Option<bool>("CanSolo", "Can solo boss?", "Can you solo Eternal Dragon?", false),
```

Comment:
add in the description that tickign this off will make the user join public rooms so he could be aware of it